ckanext/recombinant/write_xls.py

```diff
@@ -2,12 +2,13 @@
 
 from ckanext.recombinant.plugins import get_table
 from ckanext.recombinant.errors import RecombinantException
+from datatypes import data_store_type
```

Comment:
better to use the complete path `from ckanext.recombinant.datatypes ...` and not rely on relative imports